src/components/settings/components/Relay.js

```diff
@@ -44,30 +45,29 @@ const RealySettingForm = ({
     },
   };
 
-
   return (
     <div className="" >
       <Form layout="horizontal" className="p15">
         <Form.Item label="Choose Relay" colon={false}>
           {form.getFieldDecorator('relay', {
-            initialValue:2,
+            initialValue:relayConfig.value,
             rules:[]
           })(
-            <Radio.Group className="">
+            <Radio.Group className="" onChange={handleChange}>
               {
-                [1,2,3].map((item,index)=>
-                  <Radio className="d-flex align-items-center mb15 w-100" value={item} key={index}>
+                relay.nodes.map((item,index)=>
+                  <Radio className="d-flex align-items-center mb15 w-100" value={item.value} key={index}>
                     <div className="ml10">
                       <div className="row align-items-center no-gutters">
                         <div className="col-7 mr10">
-                          <Input size="large" value="Default Loopring Relay"  />
+                          <Input size="large" value={item.name}  disabled/>
                         </div>
                         <div className="col mr10">
-                          <Input size="large" value="27.0.0.01" />
+                          <Input size="large" value={item.value} disabled/>
                         </div>
                         <div className="col-auto">
-                          { index >0 &&
-                            <a href="" onClick={gotoEdit} className="">Edit</a>
+                          { item.custom &&
+                            <a href="javascript:void(0)" onClick={gotoEdit.bind(this, item.id)} className="">Edit</a>
```

Comment:
use e.preventDefault() to fix this window default actions

Reply:
fixed